samples/apps/copilot-chat-app/webapp/src/components/chat/chat-history/ChatHistoryItem.tsx

```diff
@@ -91,6 +101,18 @@ export const ChatHistoryItem: React.FC<ChatHistoryItemProps> = ({ message, getRe
         content = <PlanViewer message={message} messageIndex={messageIndex} getResponse={getResponse} />;
     } else if (message.type === ChatMessageType.Document) {
         content = <ChatHistoryDocumentContent isMe={isMe} message={message} />;
+    } else if (message.content.startsWith('data:image')) {
```

Comment:
I see us doing this "hack" check in a few places. it seems like we should add an `Image` entry into `ChatMessageType` to be consistent with how other content types are rendered.

Reply:
This is no longer relevant - only treating uploaded images as docs